docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -530,20 +530,62 @@ To use this library, use the procedure described in this section.
 
     Gradle:
 
-      ```groovy
-      dependencies {
-          compile 'org.zowe.apiml.sdk:zaas-client:{{version}}'
-      }
+      1. Create a `gradle.properties` file in the root of your project if one does not already exist.
+
+      2. In the `gradle.properties` file, set the URL of the specific Artifactory containing the _SpringEnabler_ artifact.
+
       ```
+      # Repository URL for getting the enabler-java artifact
+      artifactoryMavenRepo=https://zowe.jfrog.io/zowe/libs-release/
+      ```
+
+      3. Add the following _Gradle_ code block to the `repositories` section of your `build.gradle` file:
+
+        ```gradle
+        repositories {
+            ...
+
+            maven {
+                url artifactoryMavenRepo
+            }
+        }
+        ```
+
+      4. Add the following _Gradle_ dependency:
+
+        ```groovy
+        dependencies {
+            compile 'org.zowe.apiml.sdk:zaas-client:{{version}}'
+        }
+        ```
 
     Maven:
 
-      ```xml
-      <dependency>
-                  <groupId>org.zowe.apiml.sdk:zaas-client</groupId>
-                  <artifactId>{{version}}</artifactId>
-      </dependency>
-      ```
+      1. Add the following _XML_ tags within the newly created `pom.xml` file:
+
+        ```xml
+        <repositories>
+            <repository>
+                <id>libs-release</id>
+                <name>libs-release</name>
+                <url>https://zowe.jfrog.io/zowe/libs-release/</url>
+                <snapshots>
+                    <enabled>false</enabled>
+                </snapshots>
+            </repository>
+        </repositories>
+        ```
+    
+        **Tip:** If you want to use snapshot version, replace libs-release with libs-snapshot in the repository url and change snapshots->enabled to true.
+      
+      2. Then add the following _Maven_ dependency:
+    
+        ```xml
+        <dependency>
+                    <groupId>org.zowe.apiml.sdk:zaas-client</groupId>
+                    <artifactId>{{version}}</artifactId>
```

Comment:
This still doesn't seem to be right.  It needs to be something more like:
```
    <dependency>
      <groupId>org.zowe.apiml.sdk</groupId>
      <artifactId>zaas-client</artifactId>
      <version>{{version}}</version>
    </dependency>
```
You should mention the format of the version, e.g. "1.22.0".

It would also be helpful to include the imports and the class statements with extends/implements keywords.  Maybe you could point to a fully functioning example .java file and include this in your regular builds so it would always work.

Reply:
@nosrednayduj thanks for the review, I missed fixing the artifact ID. I've also clarified the versioning follows semantic versioning format, and added `import` statements to the `SampleZaasClientImplementation` example.

At the moment none of the internal services of the API Mediation Layer implement the Zaas Client for authentication, it is just for extenders to use. So we don't have anything to point to as a legitimate example now, and I don't think it would add significant value to add the `SampleZaasClientImplementation` as a standalone file.

Reply:
Thank you!  Looks good.